File: eccorce_app/app/src/main/java/com/example/eccomerce_app/dto/request/AddressRequestUpdateDto.kt
package com.example.eccomerce_app.dto.request

import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class AddressRequestUpdateDto(
    @Serializable(with = UUIDKserialize::class)
    val id: UUID?,
    val longitude: Double?,
    val latitude: Double?,
    val title: String?
)

File: ecommerc_dotnet/dto/Request/AddressRequestUpdateDto.cs
using Microsoft.Build.Framework;

namespace ecommerc_dotnet.dto.Request;

public class AddressRequestUpdateDto
{
    [Required]public Guid id { get; set; } 
    public decimal? longitude { get; set; } = null;
     public decimal? latitude { get; set; } = null;
     public string? title { get; set; } = null;
}